crates/node/src/lib.rs

```diff
@@ -77,6 +77,19 @@ pub async fn start(config: NodeConfig) -> eyre::Result<()> {
 
     let mut stdin = tokio::io::BufReader::new(tokio::io::stdin()).lines();
 
+    match network_client
+        .subscribe(IdentTopic::new(
+            "9df3h13HTCTgEoEnX77VaSoWrVkNxFiQmtuVGLorjycX".to_string(),
```

Comment:
where's this derived from?

Reply:
Check description

Reply:
Understood, but then we can use non-valid context IDs here, what do you think?

Reply:
Updated